lib/src/screens/exchange/exchange_page.dart

```diff
@@ -223,6 +223,15 @@ class ExchangePage extends BasePage {
                                     type: exchangeViewModel.wallet.type),
                                 addressTextFieldValidator: AddressValidator(
```

Comment:
Better to make new validator which will use address validator and domain validator (create if not exists) and keep validation for address fields.